README.md

```diff
@@ -169,30 +174,31 @@ docker service create \
     --name amivapi-cron --network backend \
     --config source=amivapi_config,target=/api/config.py \
     amiveth/amivapi amivapi cron --continuous
+
+# To attach your command line to the docker instance, use
+docker attach amivapi... # Use tab completion to find the name of the service
```

Comment:
Thinking about it: This is not at all how one should use a docker service, and not really required for the amivapi, so I don't think we should recommend this here.